### ISSUE TITLE
Refactor "Pathway or Network prediction"

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/domain-ontology#75]
http://edamontology.org/operation_3439 (Pathway or network prediction) is now obsolete in EDAM.

_Originally posted by @allysonlister in https://github.com/FAIRsharing/domain-ontology/issues/73#issuecomment-769198121_

**Comment by allysonlister** [comment]
As described in the latest release of EDAM (you can see it at http://www.ontobee.org/ontology/EDAM?iri=http%3A%2F%2Fedamontology.org%2Foperation_3439), "Notions of pathway and network were mixed up, EDAM 1.24 disentangles them." EDAM suggests one of: Protein interaction network prediction; Metabolic pathway prediction; Gene regulatory network prediction

**Comment by allysonlister** [comment]
Although EDAM suggests the three classes mentioned in the previous comment, I find them to be far too specific to be useful as replacements for "pathway or network prediction". Rather, I suggest we replace this class with the more generic "Prediction and Recognition", as all if its child classes are too specific:
![predictrecog](https://user-images.githubusercontent.com/143586/106189932-d4560180-61a0-11eb-90d7-5d7ae0289c19.png)


There are currently [21](https://fairsharing.org/search/?q=&content=standard&content=biodbcore&content=policy&content=collection&name=&taxonomies=&organisations=&shortname=&description=&supportlinks=&licenses=&countries=&maintainers=&expanded_onto_domains=pathway+or+network+prediction&expanded_onto_disciplines=&user_defined_tags=&search_state=visible&selected_facets=expanded_onto_domains_exact:Pathway%20or%20network%20prediction) records annotated with this term, and I think this refactoring would work for them all. However, I also think that we should have a FAIRsharing preferred label so as not to confuse the EDAM prediction and recognition with more generic concepts. The EDAM definition is:

> Predict, recognise, detect or identify some properties of a biomolecule.

And the term label implies something more generic than specifically about a biomolecule.

So, in summary:

* refactor from http://edamontology.org/operation_3439 (pathway or network prediction) to http://edamontology.org/operation_2423 (prediction and recognition)
* operation_2423 is already visible in DRAO
* update preferred label of operation_2423 from it's current FAIRsharing value of "Computational predictions" to "Computational biological predictions"

I'm really not sure about the third point though. Please would @delphinedauga and @Drosophilic provide their opinions on this suggestion as a whole, and the label specifically?

Thanks!

**Comment by Drosophilic** [comment]
Hi @allysonlister - this all sounds good. I'm not sure what you mean about your third point (operation_2423 is already visible in DRAO), but I follow your summary and it sounds the appropriate course of action.

**Comment by delphinedauga** [comment]
Hi @allysonlister ! 
I think it's a pity to lose "pathway or network prediction". One term in DRAO which is not too far away is "network model", but it's not really adapted for biology.

**Comment by allysonlister** [comment]
Thanks very much both of you!

> Hi @allysonlister - this all sounds good. I'm not sure what you mean about your third point (operation_2423 is already visible in DRAO), but I follow your summary and it sounds the appropriate course of action.

The term suggested by EDAM as a possible replacement, http://edamontology.org/operation_2423 ("Prediction and recognition") already exists in DRAO with the FAIRsharing label of "Computational predictions". As such, this term (operation_2423) is already accessible to curation right now. This means I can re-curate the existing records now, in concert with the changes to DRAO.

Thanks again - I'll get started both on this and #74.

**Comment by allysonlister** [comment]
Sorry @delphinedauga I hadn't seen the second half of your comment. Here are my thoughts on your suggestion for adding things:

- "Protein interaction network prediction": we already have both "protein interactions" (visible) and "protein interaction prediction" (not currently visible). Would either of these work? We could always make "Protein interaction prediction" visible. However, if you would prefer "Protein interaction network prediction", let me know. 
- "Metabolic pathway prediction": we don't have this or similar ones. http://edamontology.org/operation_3929 - happy to add it if you would like?
- "Gene regulatory network prediction": we don't have this or similar ones.  http://edamontology.org/operation_2437 - happy to add it if you would like?

**Comment by allysonlister** [comment]
I've done all of the work initially planned for this ticket. I'll leave it open for @delphinedauga's reply to see if we should make any further modifications.

**Comment by allysonlister** [comment]
@delphinedauga has checked this modification.
